drivers/sdhc/rcar_mmc.c

```diff
@@ -1152,7 +1245,7 @@ static int rcar_mmc_set_clk_rate(const struct device *dev,
 
 	host_io->clock = ios->clock;
 
-	LOG_INF("%s: set clock rate to %d",
+	LOG_DBG("%s: set clock rate to %d",
```

Comment:
Spurious change?

Reply:
I believe that that log level change should go into separate commit. Please remember that commits needs to be atomic.

Reply:
done